embedchain/embedchain.py

```diff
@@ -81,53 +82,104 @@ def add_local(self, data_type, content, metadata=None, config: AddConfig = None)
             metadata,
         )
 
-    def load_and_embed(self, loader: BaseLoader, chunker: BaseChunker, src, metadata=None):
+    async def upload_data(self, ids, documents, metadatas, src, metadata):
```

Comment:
upload is a weird term if you ask me, because you can also run a local database.